src/RecoveryServices/RecoveryServices.Backup.Models/CommonModels/Enums.cs

```diff
@@ -321,6 +321,15 @@ public enum RetentionScheduleFormat
         Weekly = 2
     }
 
+    /// <summary>
+    /// Represents the type of the policy
+    /// </summary>
+    public enum PolicyType
```

Comment:
I would like to recommend to rename PolicyType to PSPolicyType, so that we can know this type is from powershell side clearly (rather than SDK side)

Reply:
will do